frontend/apps/cloudstorage/ftp.lua

```diff
@@ -33,6 +35,8 @@ function Ftp:downloadFile(item, address, user, pass, path, close)
     local url = generateUrl(address, user, pass) .. item.url
     local response = FtpApi:downloadFile(url)
     if response ~= nil then
+        local util = require("util")
+        path = util.fixUtf8(path, "_")
```

Comment:
If it's not used anywhere else you should also be able to use `path = require("util").fixUtf8(path, "_")`

@houqp does it matter?